Aptoide/aptoide-client-v8
# [MOB-261-262] Installed-Installation to room

**What does this PR do?**

   This PR aims at migrating installed and installation tables to Room.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] RoomInstalled.java

**How should this be manually tested?**

Open aptoide and try to perform downloads, install apps, try updates.
Note: I did not test any of the roots flows as I dont have any rooted device. Since we are still going to change the Download table, maybe this flow (rooted) should be tested afterwards.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-261](https://aptoide.atlassian.net/browse/MOB-261)
[MOB-262](https://aptoide.atlassian.net/browse/MOB-262)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
